sensors/lidar.go

```diff
@@ -25,9 +25,9 @@ type TimedLidar interface {
 // TimedLidarReadingResponse represents a lidar reading with a time & allows the caller
 // to know if the reading is from a replay camera.
 type TimedLidarReadingResponse struct {
-	Reading        []byte
-	ReadingTime    time.Time
-	IsReplaySensor bool
+	Reading            []byte
+	ReadingTime        time.Time
+	TestIsReplaySensor bool
```

Comment:
I renamed this because this variable is solely used for testing purposes, which wasn't clear to me initially.

Reply:
[opt/nit] `TestingIsReplaySensor`